ngx-tools/type-guards/src/is-valid-date/is-valid-date.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Determine if an item is a valid date
+ *
+ *
+ * @param x - The item to test
+ * @return The result
+ *
+ * @example
+ * isValidDate('foo'); // Returns: false
+ * isValidDate('2020-02-07');    // Returns: true
+ */
```

Comment:
I _believe the JSDoc needs to be touching the function it documents for our generator to automatically tie them together. So I think we need to move the import to the top of the file